src/core/yargs.js

```diff
@@ -1,13 +1,28 @@
-import fs from 'fs';
 import yargs from 'yargs';
 import path from 'path';
+import cosmiconfig from 'cosmiconfig';
 
-function loadRCFile(optionsPath) {
-  const rcFile = optionsPath || path.resolve(process.cwd(), '.sequelizerc');
-  const rcFileResolved = path.resolve(rcFile);
-  return fs.existsSync(rcFileResolved)
-    ? JSON.parse(JSON.stringify(require(rcFileResolved)))
-    : {};
+function noExtLoader (filename, content) {
+  try {
+    return cosmiconfig.loadJson(filename, content);
+  } catch (_) {
+    return require(filename);
+  }
+}
+
+const explorer = cosmiconfig('sequelize', {
```

Comment:
If you plan to update to cosmiconfig v6, the API has changed. You should now use `cosmiconfigSync()` and `explorer.search()`

Reply:
Sure, if they will drop Node < 8 support (currently the minimal version for CLI is 6 according to package.json) since Node 8 is minimal version fro cosmiconfig.

Reply:
Hi everyone, I am currently super busy maintaining the main repo, that's why I haven't shown up around in the CLI repo, thanks for the patience (and by the way I don't think I will be showing up in the near future as well), just thought I'd say that

> if they will drop Node < 8 support

we definitely will do that... Probably drop Node < 10 actually.

Reply:
It's okay, I bet you're currently working on v6 and resolving of tones of issues in the main repo :)

> we definitely will do that... Probably drop Node < 10 actually.

Nice, then I'll move to the next major version of cosmiconfig as well, once you'll drop end-of-life Node versions.